src/components/30-organisms/modal/index.js

```diff
@@ -66,8 +67,18 @@ class AXAModal extends InlineStyles {
           <div
             class="o-modal__content ${this.forced
               ? 'o-modal__content--forced'
-              : ''}"
+              : ''} ${this.noheader ? 'o-modal__content--noheader' : ''}"
```

Comment:
This is brittle and exactly what [classMap](https://lit.dev/docs/templates/directives/#classmap) was invented for - please use it.

Reply:
Done